src/app/services/cell.service.ts

```diff
@@ -15,11 +16,17 @@ export class CellService extends GlobalService {
     super();
   }
 
-  getCells(): Observable<any> {
+  getCells(orderedBy: string= ''): Observable<any> {
```

Comment:
Better to use `None` than `''` when you have no value for a parameter.

Reply:
None give me this error : 
ERROR in src/app/services/cell.service.ts(19,31): error TS2304: Cannot find name 'None'.

You mean null I think.